README.md

```diff
@@ -24,105 +24,84 @@ It includes the following libraries:
-
-The [NCEPLIBS-external documentation](https://github.com/NOAA-EMC/NCEPLIBS-external) describes how to install the required dependencies and the NCEPLIBS on supported platforms, and how preconfigured platforms have been set up for the users. The definitions of the different support levels (e.g. pre-configured, configurable) are on the [Supported Platforms and Compilers](https://github.com/ufs-community/ufs/wiki/Supported-Platforms-and-Compilers) page. The following section provides general instructions for building the NCEPLIBS.
+| HDF5            | hdf5-1.10.4                                                |
+| NetCDF          | netcdf-c-4.7.3, netcdf-fortran-4.5.2                       |
 
-### Get and Build the Code
+ The [NCEPLIBS-external](https://github.com/NOAA-EMC/NCEPLIBS-external) project provides an convenient way to install the required TPL on supported platforms such as generic Linux/macOS systems.
 
-**Obtaining the code**
+## Obtaining, Building, Using NCEPLIBS
 
-Clone the repository and its submodules:
+### Obtaining the code
 
-`git clone -b ufs-v1.0.0 --recursive https://github.com/NOAA-EMC/NCEPLIBS`
+Clone the repository:
 
-**Building**
+`git clone https://github.com/NOAA-EMC/NCEPLIBS`
 
-CMake uses an out-of-source build, so create a directory for the build:
+### Building the libraries
 
-`mkdir build`
+`CMake` employs an out-of-source build.  Create a directory for configuring the build and cd into it:
 
-`cd build`
+`mkdir -p build && cd build`
 
-Set the compilers, if needed, to match those being used for compiling NCEPLIBS-external and the UFS applications later: `FC`, `CC`, and `CXX` environment variables can be used to point to the desired Fortran, C, and C++ compilers.
+Set the compilers, if needed, to match those being used for compiling the TPL listed above: `FC`, `CC`, and `CXX` environment variables can be used to point to the desired Fortran, C, and C++ compilers.
 
-Then, run CMake from inside your build directory and pass it the location of the NCEPLIBS-external install directory (if using).
+Execute `cmake` from inside your build directory.
 
-`cmake .. -DEXTERNAL_LIBS_DIR=<path-to-external-libs-install>`
+`cmake -DCMAKE_INSTALL_PREFIX=<nceplibs-prefix> ../NCEPLIBS`
 
-For libraries that are installed by NCEPLIBS-external, no additional settings are required for building the NCEPLIBS other than the `-DEXTERNAL_LIBS_DIR` flag (but see the note on static builds below). For libraries that were installed separately from NCEPLIBS-external, it may be necessary to set the appropriate environment variables so that CMake is able to find them. Please refer to the [NCEPLIBS-external documentation](https://github.com/NOAA-EMC/NCEPLIBS-external) for details.
+If the TPL are not located in a path recognized by `cmake` e.g. `/usr/local`, it may be necessary to provide the appropriate environment variables e.g. `<package_ROOT>` or `CMAKE_PREFIX_PATH` so that `cmake` is able to locate these dependencies.
```

Comment:
I assume that for this to work, we need to make some changes to NCEPLIBS-external to provide the required cmake configuration files?

Reply:
Not sure. I don’t build TPL with the externals

Reply:
> Not sure. I don’t build TPL with the externals

We'll take care of this.